src/pages/scheduler/components/share/__tests__/ShareLinkOptions.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+
+import { ShareLinkOptions } from '../ShareLinkOptions';
+
+const slug: string = 'xGIvxVgsvCYa';
+
+describe('ShareLinkOptions', () => {
+  describe('when loading', () => {
+    it('should not render any display link', () => {
+      const { container } = render(<ShareLinkOptions slug={slug} isSmallScreen={false} loading={true} />);
+      expect(container).not.toHaveTextContent('/s/xGIvxVgsvCYa');
+    });
+
+    it('should display `Loading...` message', () => {
+      render(<ShareLinkOptions slug={slug} isSmallScreen={false} loading={true} />);
+      expect(screen.getByDisplayValue('Loading...')).toBeInTheDocument();
+    });
+  });
+
+  describe('when sending a slug', () => {
+    it('should correctly render the display link', () => {
+      render(<ShareLinkOptions slug={slug} isSmallScreen={false} loading={false} />);
+      expect(screen.getByDisplayValue('https://' + window.location.hostname + '/s/xGIvxVgsvCYa')).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByText('/.*xGIvxVgsvCYa/')).toBeInTheDocument();
```
using regex simplifies this a bit